cms/envs/production.py

```diff
@@ -17,10 +17,12 @@
 from django.urls import reverse_lazy
 from path import Path as path
 
+from edx_django_utils.plugins import plugin_settings
+
+from openedx.core.djangoapps.plugins.constants import ProjectType, SettingsType
```

Comment:
Nit: I’m not really clear on why this moved and why there are so many new lines.

Reply:
moving it was required by quality checks and I added newlines to separate unrelated things. I'm not particularly that attached to new line on line 23, but addition of 21 makes sense to separate the locations of those to imports

Reply:
Nit: The newline for 21 makes sense, but I don't think the newline on line 19 is needed.  I don't think we need to separate edx org dependencies from other third-party dependencies.